csharp/extractor/Semmle.Extraction.CIL.Driver/ExtractorOptions.cs

```diff
@@ -81,15 +79,14 @@ public AssemblyInfo(string path)
                     // This failed on one of the Roslyn tests that includes
                     // a deliberately malformed assembly.
                     // In this case, we just skip the extraction of this assembly.
-                    isAssembly = false;
+                    throw new InvalidAssemblyException();
```

Comment:
Perhaps document the method and mention that it may throw an `InvalidAssemblyException`.